telemetry/definitions/commonDefinitions.json

```diff
@@ -4995,6 +5010,48 @@
                 }
             ]
         },
+        {
+            "name": "cwlLiveTail_Start",
```

Comment:
I am certain that we have existing cloudwatch metrics. Yet these new "live tail" metrics don't use the same prefix, so now both will be harder to discover.

Reply:
my bad. I perceived this as a service on it's own. could we rename it to `cloudwatchlogs_Start`. 
if needed later we could add field distinguishing as `type:liveTail`

Reply:
> could we rename it to `cloudwatchlogs_Start`.

I think `cloudwatchlogs_startLiveTail` would be fine. As long as the same prefix is used as other CWL metrics.